roles/ceph-common/tasks/main.yml

```diff
@@ -67,6 +78,12 @@
     is_ceph_infernalis={{ (ceph_stable and ceph_stable_release not in ceph_stable_releases) or (ceph_dev) or (ceph_stable_rh_storage and (rh_storage_version.stdout | version_compare('0.94', '>'))) }}
 
 - set_fact:
+    is_ceph_infernalis=True
```

Comment:
Ok it makes sense. However we can't drop the multi Ceph version support all of the sudden. 
So yes I think the next big change will happen when I'll move the repo to ansible 2 so ceph-ansible 2.0 probably.